modules/solid_mechanics/test/tests/dynamics/time_integration/tests

```diff
@@ -72,4 +72,13 @@
     design = "Dynamics.md"
     issues = "#18687"
   []
+
+  [direct_central_difference]
+    type = 'Exodiff'
+    input = 'direct_central_difference.i'
+    exodiff = 'direct_central_difference_out.e'
+    abs_zero = 1e-09
+    requirement = "The solid mechanics app shall support a generalized form of central difference time integration"
```

Comment:
I think you should be more descriptive than "generalized", communicating what this new time integrator does that the framework integrator does not

Reply:
Fair point, Ill change that

Reply:
[d9b75bb](https://github.com/idaholab/moose/pull/28175/commits/d9b75bb14970b3f38865683f49e6bac2fd62edc1)